### PR TITLE
Negar/embed chart

### PR DESCRIPTION
**Pull request by negar-binary**
replace livechart with highchart
remove livechart page and unneeded livechart code
add tradingview to footer
(deployed to qa17)

### CODE REVIEW

**Comment by raunakkathuria** [comment]
@negar-binary just a note for future, use descriptive message as commit message

File: src/javascript/binary/chart.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+var chart;
```
Comment (raunakkathuria) [inline src/javascript/binary/chart.js]: why these extra file for these variables only?

Comment (negar-binary) [inline src/javascript/binary/chart.js]: @raunakkathuria there was a file called livechart and I removed everything else in it and changed the name to chart, but I think we need these vars as they are used elsewhere. should I move them to another file?

**Comment by raunakkathuria** [comment]
@borisyankov can you also review this, I am reviewing backend code for this

File: src/javascript/binary/chart.js (hunk 1 above)
Comment (borisyankov) [inline src/javascript/binary/chart.js]: Yeah, certainly need to be declared wherever they are used.

Comment (negar-binary) [inline src/javascript/binary/chart.js]: @borisyankov @raunakkathuria done

**Pull request closed without merging**